src/server/master_server/master_metadata_service_impl.cc

```diff
@@ -1,7 +1,6 @@
-#include "src/server/master_server/master_metadata_service_impl.h"
-
 #include "grpcpp/grpcpp.h"
 #include "src/protos/grpc/master_metadata_service.grpc.pb.h"
+#include "src/server/master_server/master_metadata_service_impl.h"
```

Comment:
also explicitly include "src/common/utils.h" as you are using `ConvertProtobufStatusToGrpcStatus`

Reply:
Sounds good. Added.